go.mod

```diff
@@ -16,5 +16,48 @@ require (
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.6.1
 	github.com/theckman/httpforwarded v0.4.0
+)
+
+require (
```

Comment:
Are all the indirect entries required?

Reply:
I'm not sure what entries can be ommited. So that's what `go mod tidy` generates.